README.md

```diff
@@ -39,7 +39,7 @@ Follow these steps to get Dynamodb running locally
 1. **Load data into your local Dynamodb tables**
     1. Fridge Data: `sam local invoke LoadFridgeDataFunction --parameter-overrides ParameterKey=Environment,ParameterValue=local --docker-network cfm-network`
 1. **Get data from your local Dynamodb tables**
-    1. `aws dynamodb scan --table-name fridge --endpoint-url http://localhost:8000`
+    1. `aws dynamodb scan --table-name fridge --endpoint-url http://localhost:4566`
 
 ## API
```

Comment:
Can we add a section here for the Image API? The instructions you wrote on the PR should be enough :) 

1. Start local SAM API
    ```
    sam local start-api --parameter-overrides ParameterKey=Environment,ParameterValue=local --docker-network cfm-network
    ```
1. Upload image (replace `<file-path>` with your actual image path like `"@/home/user/Downloads/sample.webp"`)
    ```
    curl --request POST \
      --url http://localhost:3000/v1/image \
      --header 'Content-Type: image/webp' \
      --data-binary "@<file-path>"
    ```

Reply:
Good call! Done in 95d4a93dcb45142784f5f4308429fe9e1dbad8ee :+1: